cypher-language/build.gradle

```diff
@@ -0,0 +1,107 @@
+import ca.coglinc.gradle.plugins.javacc.CompileJavaccTask
+
+group "org.polypheny"
+
+configurations {
+    javacc
+    tests {
+        extendsFrom testRuntimeOnly
+    }
+}
+
+
+buildscript {
+    dependencies {
+        classpath group: "gradle.plugin.ca.coglinc2", name: "javacc-gradle-plugin", version: javacc_plugin_version
+    }
+}
+
+
+dependencies {
+    implementation project(":core")
+    implementation project(":sql-language")
```

Comment:
Dependencies between languages might be problematic.

Reply:
Removed dependency, as it was not longer needed.